Automattic/newspack-newsletters
# epic: consolidate newsletter editor data and UI

### All Submissions:






### Changes proposed in this Pull Request:



This PR contains changes from #1629, #1630, #1631, and #1632 which were already tested and code reviewed, so it should only require a quick review of the changeset instead of a full code review.

### How to test the changes in this Pull Request:

If anything, just ensure that the newsletter editor loads correctly and lets you select/save sender and send list info without any fatals.

### Other information:
